CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # Release History
 
 ## 1.1.0-beta.1 (Unreleased)
+### New Features
+- Public preview version of a new set of APIs to simplify the experience using IoT Plug and Play:
```

Comment:
should we mention the header file which these are contained in?
`az_iot_pnp_client.h`

Reply:
and maybe the cmake target?
`az::iot::pnp`

Reply:
added the reference to header file. not sure what do you mean by the other. can you show me what would you add on that?

Reply:
Yup here is the section in the CMake which defines that:
https://github.com/Azure/azure-sdk-for-c/blob/3d9f1a3d530ef5fa12cf481ce7e6a4b798e286d5/sdk/src/azure/iot/CMakeLists.txt#L52-L73

You can see at the very end where it defines the target name `az::iot::pnp` which is what someone would link to if they had a CMake project ingesting these.